ddl/db_integration_test.go

```diff
@@ -583,6 +587,10 @@ func (s *testIntegrationSuite5) TestErrnoErrorCode(c *C) {
 	tk.MustGetErrCode(sql, errno.ErrWrongDBName)
 	sql = "alter table test_error_code_succ modify t.c1 bigint"
 	tk.MustGetErrCode(sql, errno.ErrWrongTableName)
+	sql = "alter table test_error_code_succ change c1 _tidb_rowid bigint"
+	tk.MustGetErrCode(sql, errno.ErrWrongColumnName)
+	sql = "alter table test_error_code_succ rename column c1 to _tidb_rowid"
```

Comment:
What about `alter table add column`?

```sql
alter table t add column _tidb_rowid int;
```

Reply:
It's at line 538

Reply:
I think there is one? https://github.com/pingcap/tidb/pull/19418/files#diff-703ae6b7872b425273d1832c198598c8R538

Reply:
Sorry. I didn't notice that.